### PR TITLE
Retired `plotutils` in favor of `gnuplot`, and tested the latter on Linux&Mac.

### PR DESCRIPTION
**Pull request by dkorolev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -359,9 +358,10 @@ const std::string result = GNUPlot()
       p(cos(phi), sin(phi), bricks::strings::Printf("P%d", i));
     }
   }).AsLabels())
-  .OutputFormat("svg");
+  .ImageSize(image_dim)
+  .OutputFormat("svg");  // Although the one below is actually a "png".
 ```
-![](https://raw.githubusercontent.com/dkorolev/Bricks/plot_labels/graph/golden/labels.png)
+![](https://raw.githubusercontent.com/dkorolev/Bricks/master/graph/golden/labels-Linux.png)
```
Comment (deathbaba) [inline README.md]: KnowSheet bricks references dkorolev's bricks. Is it by design?

File: graph/docu/docu_04graph_03.md [hunk 2]
```diff
@@ -0,0 +1 @@
+![](https://raw.githubusercontent.com/dkorolev/Bricks/master/graph/golden/love-Linux.png)
```
Comment (deathbaba) [inline graph/docu/docu_04graph_03.md]: The same as above

Comment (dkorolev) [inline graph/docu/docu_04graph_03.md]: Yes, it's to have the `README.md` file readable during the review. Good to fix and merge?

File: graph/docu/docu_04graph_04.cc [hunk 3]
```diff
@@ -39,8 +39,18 @@ SOFTWARE.
 TEST(Graph, GNUPlotScience) {
   // Where visualization meets science.
   using namespace bricks::gnuplot;
-const char* const extensions[2] = { "svg", "png" };
+#ifndef BRICKS_APPLE
+const char* const formats[2] = { "dumb", "pngcairo" };
```
Comment (deathbaba) [inline graph/docu/docu_04graph_04.cc]: What is the difference between pngcairo and png?
What is bad about using png everywhere?

File: graph/docu/docu_04graph_04.cc [hunk 4]
```diff
@@ -60,15 +70,14 @@ for (size_t e = 0; e < 2; ++e) {
...
+.OutputFormat(formats[e]);
 #else
-    .OutputFormat("svg");
-#endif
-if (FLAGS_regenerate_golden_graphs) bricks::FileSystem::WriteStringToFile(result, (std::string("golden/gnuplot.") + extensions[e]).c_str());
-#ifndef BRICKS_APPLE // TODO(dkorolev): Figure out how to run this test on Apple.
-if (!e) ASSERT_EQ(result, bricks::FileSystem::ReadFileAsString(std::string("golden/gnuplot.") + extensions[e]));
+    .OutputFormat("svg");  // Although the one below is actually a "png".
 #endif
+if (FLAGS_regenerate_golden_graphs) bricks::FileSystem::WriteStringToFile(result, ("golden/science-" + BRICKS_ARCH_UNAME + '.' + extensions[e]).c_str());
```
Comment (deathbaba) [inline graph/docu/docu_04graph_04.cc]: These lines looks too long. Have you indented them? May be add bracers?

File: graph/docu/docu_04graph_04.cc (hunk 3 above)
Comment (dkorolev) [inline graph/docu/docu_04graph_04.cc]: It's a slightly smoother looking `png`.

File: graph/docu/docu_04graph_04.cc (hunk 4 above)
Comment (dkorolev) [inline graph/docu/docu_04graph_04.cc]: In the `docu_*` file indenting this line would result in it making it into `README.md`, which is undesirable.

I could break it into several unindented lines though, if you think it's worth it.

**Pull request merged**